#request fields
Header header
arm_navigation_msgs/MotionPlanRequest motion_plan_request
geometry_msgs/Pose[] poses
int8 type
#request constants
int8 SIMPLE_IK=0
int8 LINE=1
int8 CIRCLE=2
int8 PATH=3
---
# A solution trajectory, if found
arm_navigation_msgs/RobotTrajectory trajectory

# Error code - encodes the overall reason for failure
arm_navigation_msgs/ArmNavigationErrorCodes error_code

# More detailed error codes (optional) - encode information about each point in the returned trajectory
arm_navigation_msgs/ArmNavigationErrorCodes[] trajectory_error_codes